http.postifxTemplates

```diff
@@ -0,0 +1,10 @@
+## for http development ##
+.handler : generate method for HandleFunc
```

Comment:
This example would work as a `GoLand` live template (https://www.jetbrains.com/help/go/using-live-templates.html). Do you have a good reason for including it in `custom-postfix-templates` instead? I'd consider adding it to `custom-postfix-templates` if there is an advantage.

Let me know if you're not familiar with how the live templates work.

Reply:
@philipgriggs , 

Thanks for your comments. I also have  some live templates with postfix templates format for my coding habit. But I think you opinion make sense to keep clear postfix templates. I delete them from this repository,  only just keep in my user-defined postfix.